src/pages/Restaurants/RestaurantsPage.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { CustomCardsSection, Navbar, Footer, RestaurantsHeader } from "../../components";
+import { RestaurantsData } from "../../data";
+import "./RestaurantsPage.scss"; // Import SCSS file
+
+const Restaurants = () => {
+  return (
+    <>
+      <Navbar />
```

Comment:
If the navbar will be on every page, you can simply place it once in the App.tsx file.